client/gatsby-node.js

```diff
@@ -36,6 +35,13 @@ exports.onCreateNode = function onCreateNode({ node, actions, getNode }) {
       createNodeField({ node, name: 'component', value: component });
```

Comment:
GitHub's UI doesn't let me put the comment in the right place, sadly, but we shouldn't be generating any MarkdownRemark nodes, so I think this can go.